adapter/botservice.py

```diff
@@ -10,9 +10,9 @@ class BotAdapter:
     def get_queue_info(self): ...
     """获取内部队列"""
 
-    def __init__(self, session_id: str = "unknown"):
+    def __init__(self, session_id: str = "unknown", current_model: str = "default"):
         self.supported_models = []
-        self.current_model = "default"
+        self.current_model = current_model
```

Comment:
I'm afraid you have misunderstood the usage of current_model. It refers to the code name of the model that we send to the backend server. If you want to know what kind of adapter we're using, you can find the answer in the ConversationContext.